.github/workflows/gpu-ci.yml

```diff
@@ -89,6 +89,8 @@ jobs:
           export PATH=$CONDA_PREFIX/bin:$PATH
           export FF_HOME=$(pwd)
           export FF_USE_PREBUILT_LEGION=OFF #remove this after fixing python path issue in Legion
+          curl https://sh.rustup.rs -sSf | sh -s -- -y
```

Comment:
can we install Rust through the conda environment file, and modify the docker builds so that the flexflow-environment machine has it pre-installed?